### PR TITLE
Activities for comments

### PR DESCRIPTION
**Pull request by nickvergessen**
- Adding a comment triggers an activity
- There is a filter in the activity app to show only comment activities
- Comments are shorten in the sidebar activity stream to avoid problems and keep an overview.

@blizzz 
@PVince81 which issue holds this task?
### Activity page

activity_-_owncloud_-_2016-02-05_11 32 16
### Sidebar

files_-_owncloud_-_2016-02-05_11 32 57

### CODE REVIEW

**Comment by PVince81** [comment]
@nickvergessen it was just a checkbox, I added the ref here https://github.com/owncloud/core/issues/16328#issuecomment-179120807

**Comment by PVince81** [comment]
Tested, works :+1: (also with shared files)

File: lib/private/comments/manager.php [hunk 1]
```diff
@@ -415,6 +430,19 @@ public function delete($id) {
...
 		}
 
+		try {
+			$comment = $this->get($id);
+		} catch (\Exception $e) {
+			// Ignore exceptions, we just don't fire a hook then
+			$comment = null;
+		}
+		if ($comment instanceof IComment) {
+			$this->dispatcher->dispatch(CommentsEvent::EVENT_DELETE, new CommentsEvent(
```
Comment (blizzz) [inline lib/private/comments/manager.php]: Maybe only fetch the Comment here, and trigger this only when the DB action succeeded?

Comment (nickvergessen) [inline lib/private/comments/manager.php]: Hmm kind of makes sense, so this is a post event as well.
Will fix

**Comment by blizzz** [comment]
Tested and works, aside of https://github.com/owncloud/core/pull/22152#discussion_r52005839 for consideration, :+1:

**Comment by blizzz** [comment]
My :+1: stays also with the additional commit

**Pull request merged**